docs/reference/commandline/start.md

```diff
@@ -45,7 +53,12 @@ $ docker start -a my_nginx_container
 ...
 ```
 
-### Start a contianer and restore from a checkpoint
+### <a name="checkpoint"></a> Start a container and restore from a checkpoint (--checkpoint) (experimental)
+
+The `--checkpoint` option is an experimental feature, and should not be
+considered stable. To read about experimental daemon options and how to enable
+them, see
+[Daemon configuration file](https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file).
```

Comment:
Hm.. I thought we had a `/go/` redirect for this one, but looks like that's only for experimental CLI features (but those are now enabled by default). https://github.com/docker/docs/blob/8d81583dcf89cf302420ca5f5c8a39b22a7e73fb/_redirects.yml#L106-L107


Perhaps we should repurpose that redirect for both cli (no longer needed) and engine features (and have a canonical section about enabling experimental features, etc).

@dvdksn perhaps you have ideas on that?